src/ibc_api/utils.py

```diff
@@ -381,18 +400,19 @@ def _download_file(src_file, dst_file, connector):
         return dst_file
     else:
         print(f"File {dst_file} already exists, skipping download.")
-
-        return []
+        return dst_file
 
 
-def download_data(db, save_to=None):
+def download_data(db, num_jobs=2, save_to=None):
     """Download the files in a (filtered) dataframe.
 
     Parameters
     ----------
     db : pandas.DataFrame
         dataframe with information about files in the dataset, ideally a subset
         of the full dataset
+    num_jobs : int, optional
+        number of parallel jobs to run, by default 2
```

Comment:
```suggestion
        number of parallel jobs to run, by default 2. -1 would use all the CPUs. 
        See: https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html
```

Reply:
Yeah, I was using this option at first... the thing is, when testing in small computers (not clusters) and downloading large files (raw bold files), using all possible cpus causes the program to crash some times. This is due to a lot of memory usage, caused by the use of "threading" as backend option in joblib, but we need it because all the parallel jobs need access to the token... 
So I was thinking using a small number of jobs as default is better, and if you're downloading smaller files like contrast maps you can always go higher, what do you think?

Reply:
Yes, I agree. I was just suggesting mentioning the -1 thing and linking joblib's docs for users to know other options